code/bit_manipulation/src/count_set_bit/brian_Kernighan_Algorithm.cpp

```diff
@@ -0,0 +1,29 @@
+//Brian Kernighan’s Algorithm. This programme uses O(logn) to count set bits.
+//Part of Cosmos by OpenGenus Foundation 
+#include <bits/stdc++.h> 
```

Comment:
#1915. Also move this file to its own directory. Delete the binary file as well.